From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jelle Raaijmakers <jelle@gmta.nl>
Date: Mon, 24 Apr 2023 15:41:00 +0200
Subject: [PATCH] SamTFE: Force __linux__ macro to be 1

We are compatible with the Linux version, and setting this macro allows
us to identify as Linux and compile successfully.
---
 SamTFE/Sources/Engine/Base/Base.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/SamTFE/Sources/Engine/Base/Base.h b/SamTFE/Sources/Engine/Base/Base.h
index 0f021cf08cba55b3f9ce8e99e23cba791b81229c..7e7ddc60b1a409aa76deddf491a67a0f35cf6dc0 100644
--- a/SamTFE/Sources/Engine/Base/Base.h
+++ b/SamTFE/Sources/Engine/Base/Base.h
@@ -21,6 +21,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
  */
 // be a little more discerning, using these macros will ensure that if someone
 // wants to use MINGW then they can
+#define __linux__ 1
 #if (defined _WIN32) || (defined _WIN64) 
   #ifndef PLATFORM_WIN32
     #define PLATFORM_WIN32 1
